packages/lib/shared/components/tooltips/apr-tooltip/MainAprTooltip.tsx

```diff
@@ -138,12 +138,15 @@ function MainAprTooltip({
   id,
   ...props
 }: Props) {
-  const canBeNegative = isQuantAmmPool(pool.type)
-  const aprToShow = apr || getTotalAprLabel(pool.dynamicData.aprItems, vebalBoost, canBeNegative)
-  const isAprNegative = bn(aprToShow).lt(0)
+  const aprToShow = apr || getTotalAprLabel(pool.dynamicData.aprItems, vebalBoost, true)
+  const isAprNegative = bn(aprToShow.replace('%', '')).lt(0)
 
   // hoverColor here is used for the text and therefore can use a semantic token
-  const hoverColor = isLBP(pool.type) ? 'inherit' : isAprNegative ? 'font.error' : 'font.highlight'
+  const hoverColor = isLBP(pool.type)
+    ? 'inherit'
+    : isAprNegative
+      ? 'font.warning'
+      : 'font.highlight'
```

Comment:
curious why the change from `font.error` to `font.warning` ? kinda seem like warning too close to default tan 

<img width="1053" height="358" alt="image" src="https://github.com/user-attachments/assets/c34d7c0b-d106-4e27-b765-7925a6e3c297" />

Reply:
It comes from the Slack discussion

<img width="814" height="380" alt="Screenshot 2025-08-14 at 10 00 20" src="https://github.com/user-attachments/assets/0ba8d573-b869-4100-a242-f0a756a4b46e" />

The idea is that "In general, I try to keep font.error for situations where the user can't act. In this case, the user can still interact with the pool". The main problem was that we were not showing negative numbers at all though.